Entities/Industry/Drill/Drill.as

```diff
@@ -301,7 +307,7 @@ void onTick(CBlob@ this)
 
 								if (isServer())
 								{
-									if (int(heat) > heat_max * 0.7f) // are we at high heat? more damamge!
+									if (heat_max - int(heat) < high_damage_window) // are we at high heat? more damamge!
```

Comment:
`heat_max - int(heat) < high_damage_window`  
is equivalent to  
`int(heat) >= heat_max - high_damage_window`  
which is a lot more readable

Reply:
... or alternatively replace `high_damage_window` into a `high_damage_threshold` which is equal to this `heat_max - high_damage_window` value

Reply:
done the 1st